### PR TITLE
Feature/order components

### PR DESCRIPTION
**Pull request by MantSingh**
WORKED ON search input and order listing item

### CODE REVIEW

File: assets/images/camera_alt.png
Comment (kartikDeloitte) [inline assets/images/camera_alt.png:1]: use svg

File: assets/images/search.png
Comment (kartikDeloitte) [inline assets/images/search.png:1]: use svg

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        placeholder={placeholder}
+        value={value}
+        onChangeText={text => onChangeText(text)}
+      />
+      <TouchableOpacity
+        activeOpacity={1}
+        onPress={() => {
+          setModalVisible(true);
+        }}>
+        <Image
```
Comment (kartikDeloitte) [inline app/components/searchInputWithCamera/searchInputWithCamera.tsx]: use svg

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const SearchInputWithCamera = ({
+  onChangeText = () => {},
+  placeholder = 'Search by product, category...',
+  value = '',
+  setPhoto = () => {},
+}) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  return (
+    <View style={styles.container}>
+      <Image
```
Comment (kartikDeloitte) [inline app/components/searchInputWithCamera/searchInputWithCamera.tsx]: use svg

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  View,
+} from 'react-native';
+import React, {useState} from 'react';
+import {COLORS} from '@/theme/colors';
+import {heightToRatio, widthToRatio} from '@/utils/commonMethods';
+import UploadImageBottomSheet from '@/bottomSheets/uploadImageBottomSheet/UploadImageBottomSheet';
+
+const SearchInputWithCamera = ({
+  onChangeText = () => {},
+  placeholder = 'Search by product, category...',
```
Comment (kartikDeloitte) [inline app/components/searchInputWithCamera/searchInputWithCamera.tsx]: use getTranslationLabel for text

File: app/components/categoriesCard/categoriesCard.tsx [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
 import {fontConfig} from '@/theme/fonts';
-
-const CategoriesCard = ({title, imagePath, onPress}) => {
+interface CategoriesCardProps {
+  title: String;
+  imagePath: ImageProps;
+  onPress: () => void;
+}
+const CategoriesCard = ({
+  title = 'Water Heaters',
```
Comment (kartikDeloitte) [inline app/components/categoriesCard/categoriesCard.tsx:13]: use getTranslationLabel

File: app/components/categoriesCard/categoriesCard.tsx [hunk 5]
```diff
@@ -1,18 +1,26 @@
...
+  imagePath = require('../../../assets/images/demoCategorieItem.png'),
```
Comment (kartikDeloitte) [inline app/components/categoriesCard/categoriesCard.tsx:14]: use import statement or use svg where ever possible

File: app/components/categoriesCard/categoriesCard.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import React from 'react';
+import {heightToRatio, widthToRatio} from '@/utils/commonMethods';
+import {Image} from 'react-native';
+import {COLORS} from '@/theme/colors';
+import {fontConfig} from '@/theme/fonts';
+
```
Comment (kartikDeloitte) [inline app/components/categoriesCard/categoriesCard.tsx]: create interface

File: app/components/categoriesCard/categoriesCard.tsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+const CategoriesCard = ({title, imagePath, onPress}) => {
+  return (
+    <TouchableOpacity activeOpacity={0.8} style={styles.continer}>
+      <Image
+        source={require('../../../assets/images/demoCategorieItem.png')}
```
Comment (kartikDeloitte) [inline app/components/categoriesCard/categoriesCard.tsx]: use svg or use import if svg not applicable

**Review by Mayur-Deloitte16 — COMMENTED** [review]
Add colors from Colors file..

File: assets/images/camera_alt.png
Comment (MantSingh) [reply assets/images/camera_alt.png:1]: Done

File: assets/images/search.png
Comment (MantSingh) [reply assets/images/search.png:1]: Done

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx (hunk 1 above)
Comment (MantSingh) [reply app/components/searchInputWithCamera/searchInputWithCamera.tsx]: Done

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx (hunk 2 above)
Comment (MantSingh) [reply app/components/searchInputWithCamera/searchInputWithCamera.tsx]: Done

File: app/components/searchInputWithCamera/searchInputWithCamera.tsx (hunk 3 above)
Comment (MantSingh) [reply app/components/searchInputWithCamera/searchInputWithCamera.tsx]: Done

File: app/components/categoriesCard/categoriesCard.tsx (hunk 4 above)
Comment (MantSingh) [reply app/components/categoriesCard/categoriesCard.tsx:13]: anyhow this data will come from backend 
so ignoring this point

File: app/components/categoriesCard/categoriesCard.tsx (hunk 5 above)
Comment (MantSingh) [reply app/components/categoriesCard/categoriesCard.tsx:14]: anyhow this data will come from backend
so ignoring this point

File: app/components/categoriesCard/categoriesCard.tsx (hunk 7 above)
Comment (MantSingh) [reply app/components/categoriesCard/categoriesCard.tsx]: Done

File: app/components/categoriesCard/categoriesCard.tsx (hunk 6 above)
Comment (MantSingh) [reply app/components/categoriesCard/categoriesCard.tsx]: Done

**Pull request merged**